airflow/utils/file.py

```diff
@@ -324,6 +326,12 @@ def find_dag_file_paths(directory: str | pathlib.Path, safe_mode: bool) -> list[
                 continue
             if not might_contain_dag(file_path, safe_mode):
                 continue
+            if include_zip_paths and zipfile.is_zipfile(file_path):
+                with zipfile.ZipFile(file_path) as current_zip_file:
+                    for zip_info in current_zip_file.infolist():
+                        mod_name, ext = os.path.splitext(zip_info.filename)
```

Comment:
What if the `zip_info` object is a directory?

Reply:
With the new commit I guess this function change is not necessary. I just forgot to remove it.